app/Models/User.php

```diff
@@ -43,6 +43,11 @@ class User extends Authenticatable
         'email_verified_at' => 'datetime',
     ];
 
+        public function setPasswordAttribute($password)
+    {
+        $this->attributes['password'] =  bcrypt(trim($password));
```

Comment:
@sabinshrestha8 `trim` is not necessary as Laravel does it on a middleware level